piet-common/Cargo.toml

```diff
@@ -14,34 +14,36 @@ cairo = ["piet-cairo", "cairo-rs"]
 web = ["piet-web"]
 
 [dependencies]
-cfg-if = "0.1.10"
 piet = { version = "0.1.0", path = "../piet" }
 piet-cairo = { version = "0.1.0", path = "../piet-cairo", optional = true }
 piet-coregraphics = { version = "0.1.0", path = "../piet-coregraphics", optional = true }
 piet-direct2d = { version = "0.1.0", path = "../piet-direct2d", optional = true }
 piet-web = { version = "0.1.0", path = "../piet-web", optional = true }
-cairo-rs = { version = "0.8.1", default_features = false, optional = true}
 
-[target.'cfg(not(any(target_arch="wasm32", target_os="windows", target_os="macos")))'.dependencies]
+cfg-if = "0.1.10"
+cairo-rs = { version = "0.8.1", default_features = false, optional = true }
+png = { version = "0.16.1", optional = true }
+
+[target.'cfg(target_os="linux")'.dependencies]
 piet-cairo = { version = "0.1.0", path = "../piet-cairo" }
```

Comment:
we want piet-cairo to be built if the --features=cairo option is passed on macOS. Maybe we're being too fancy though, and could just get rid of this option?

Reply:
Yes the `cairo` feature still works on every platform, including Windows when using gcc.

The CI only tests cairo on Linux. If we want to ensure that cairo support is awesome then we could test it on macOS and Windows too. However my understanding has been that cairo is more of a temporary backend, so probably fine to just test it on Linux.